src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -1,8 +1,24 @@
 package core.basesyntax;
 
-/**
- * Feel free to remove this class and create your own.
- */
+import core.basesyntax.figures.Figure;
+import core.basesyntax.suppliers.FigureSupplier;
+
 public class HelloWorld {
+    public static void main(String[] args) {
+
+        Figure[] listOfArray = new Figure[6];
```

Comment:
```suggestion
        Figure[] listOfArray = new Figure[6];
```
also let's create a constant for 6